about/index.md

```diff
@@ -171,6 +171,6 @@ is also available from several companies.
 [11]: https://wiki.wxwidgets.org/
 
 
-<p style="margin-top: 2.5em; text-align: center;">
-  <a href="/downloads/" class="btn btn-lg btn-primary"><i class="fa fa-download fa-fw"></i> Download now &raquo;</a>
+<p class="my-4 text-center">
```

Comment:
Hmm, shouldn't `margin-top` be replaced with `mt-4` rather than `my-4`?

Reply:
I think I needed the bottom margin now due to other style changes with Bootstrap 4.